Developers/Writing_Good_Commits.rst

```diff
@@ -1,6 +1,31 @@
 How To Write a Good Commit Message
 ==================================
 
+Quick reference
+---------------
+
+Example of a good commit:
+
+::
+
+    setup.py: Change bears' entrypoint
+
+    This entrypoint ensures that coala discovers
+    the bears correctly.
+    It helps not writing more functions inside
+    ``coalib`` for this.
+
+    Fixes https://github.com/coala/coala/issues/5861
+
+- `setup.py: Change bears' entrypoint`: Describe the change in
+   maximum of 50 characters.
+
+- `This entrypoint.. ..for this`: Describe the reasoning of your changes
+   in maximum of 72 characters.
+
+- `Fixes https://github.com/coala/coala/issues/5861`: Mention the URL
+   of the issue it closes or fixes.
```

Comment:
Then It won't be a quick reference anymore. :( because we have the same thing written down the page. If someone wants to know the difference between closes and fixes he could just scroll down and read it.. !!